### DISCUSSION TITLE
[Feature] fluent image zooming

### DISCUSSION THREAD

**Discussion by waclaw66** [immich-app/immich#3448] [Feature Request]
### The feature

Wheel zoom ratio is too low currently, recommended
`wheelZoomRatio: 0.2`
with
`transition:scale={{ duration: 400 }}`

Thanks.

### Platform

- [ ] Server
- [X] Web
- [ ] Mobile

**Comment by Fuewburvpoa** [comment]
Was about to start same discussion, it is indeed really slow in web client.
